src/index.ts

```diff
@@ -399,6 +409,29 @@ app.post('/test', async (req, res) => {
   })
 })
 
+app.get('/node/health', async (req, res) => {
+  res.send({
+    timestamp: Date.now(),
+    heliaStatus: helia.libp2p.status
+  })
+})
+
+app.get('/node/info', async (req, res) => {
+  const { blockstoreSizeMb, datastoreSizeMb, availableSizeInMb } = getDiskUsageStats()
+  const heliaVersion = (packageJson.dependencies['helia'] || 'unknown').replaceAll('^', '')
```

Comment:
Can you extract the Helia version using Helia API? See specification https://helia.io/

Reply:
Unfortunately, I couldn't find such information in this doc. 
Therefore, I implemented getting the version this way.

Reply:
Then remove `heliaVersion` from nodeInfo response